mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -655,13 +655,9 @@ pub fn get_locked_amount(ctx: &MmArc, coin: &str) -> MmNumber {
 }
 
 /// Get number of currently running swaps
```

Comment:
fix this doc comment

Reply:
[dd5418f](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2301/commits/dd5418fc2444679a57acdb5370788b2ac1fcc1da)

Reply:
also i wanted to clear the swaps in `MmCtx::stop` instead since that's how we destruct MmCtx. but that won't work because we have no access to `SwapsContext` data type.